### ISSUE TITLE
help, i need some info / help, not just any info

### ISSUE THREAD

**Issue by sydb** [rgxg/rgxg#6]
Help! I need some info
Help! Not just any info
Help! You know I need to know
Help!

When I am working on a regexp from a range
I sometimes need to know the switches. That's not strange!
And though this tool is cool, dash H puts me in a rage;
Now I find, though it's unkind, I must open the man page!

Boy, do I want to do the rest of the song. But it's way past my bedtime. So for starters, thank you very much for this tool. Just what I was looking for.

I realize this is a 0.1 release, so not everything will be working, but (it strikes me):
1. a `-v` (or `--version` or whatever) switch is pretty important _especially_ in the 0.x.x phases, and
2. similarly a `-h` (or `-?` or `--help` or whatever) switch is really nice. I realize that `rgxg CMD -h` works, but having a `-h` that gave me the list of CMDs would be very helpful.

**Comment by hvhaugwitz** [comment]
There is `rgxg version` and `rgxg help`. Having said that I certainly can add `-h` and '-v` at top level, if that helps.

**Comment by frittentheke** [comment]
@hvhaugwitz maybe @sydb is suggesting to do it POSIX style for your args? As in ... https://www.gnu.org/software/libc/manual/html_node/Argument-Syntax.html

**Comment by sydb** [comment]
Thanks, @frittentheke. Yes, suggesting either POSIX (-h, -v) or GNU (--help or --version) or both.

When I first sat down with this (lovely) new toy, the first thing I did was type `rgxg -h`. That didn’t work, so I tried `rgxg --help`. That did’t work, so I read the man page.
On first glance, it _looks_ like the man page says -h is supposed to work. But that’s not a switch for `rgxg`, but rather for `rgxg alternation`. So I was even confused there for a moment. (My own fault for not reading carefully, but who reads carefully? : - ) Seriously, users are supposed to read documentation carefully for the intricate parts of a new command (like the cidr subcommand), but no point in making a user work just to get basic information. (I did discover, I think after I posted the original issue, that `rgxg` with no arguments does the right thing. I admit I should have tried that.)
